ipa-macros/src/derive_gate/mod.rs

```diff
@@ -1,12 +1,7 @@
-use crate::tree::Node;
+use crate::parser::*;
```

Comment:
I am surprised you got away with this. Isn't clippy configured to flag star imports?

Reply:
Apparently not 😆